beacon.yaml

```
@@ -338,10 +338,11 @@ components:
           items:
             $ref: '#/components/schemas/BeaconAlleleRequest'
         info:
-          description: 'Additional structured metadata, key-value pairs.'
-          type: array
-          items:
-            $ref: '#/components/schemas/KeyValuePair'
+          description: 'Additional unspecified metadata about the Beacon service.'
+          type: object
+          example:
+            beaconNetwork: ELIXIR Europe
```

This is confusing, we should use generic examples which do not mean anything (using "beaconNetwork" may be misleading):
```suggestion
            additionalInfoKey1: additionalInfoValue1
```